src/styles/cardStyles/index.ts

```diff
@@ -0,0 +1,12 @@
+import variables from '../variables';
+import GetCardStyles from './types';
+
+// Get card style for cardStyleInterpolator
```

Comment:
For better intellisense:
```suggestion
/** Get card style for cardStyleInterpolator */
```